### PR TITLE
React-Redux Capstone PR

### PR DESCRIPTION
**Pull request by adamteddychang**
- [Loom Link](https://www.loom.com/share/2c4ab45fc77e44b59caf53729db39f75)
- [Live Link](https://adamstocksapp.netlify.app)
- In this application:
- I used React-Redux to complete the single-page web app
- I used the Financial modeling prep API for this project
- I used the design by Nelson Sakwa on Behance.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "max-len": "off",
+    "no-param-reassign":"off",
+    "no-use-before-define":"off",
+    "react/prop-types": "off"
```
Comment (mikenath223) [inline .eslintrc.json:25]: - [ ] Kindly ensure the rules here match the given [linting rules](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Stocks App
+
+> This project utilizes Financial modeling prep API to dispolay stocks, foreign exchange, cryptos...and so on. I used React-Redux along with jest testing to complete this project. 
+
+
+![Capture1](https://user-images.githubusercontent.com/63560332/147289709-5c7989f2-0893-43da-a78f-89686f51faab.PNG)
+![Capture2](https://user-images.githubusercontent.com/63560332/147289716-2f677568-42b1-4925-876f-0cb03b145f52.PNG)
```
Comment (mikenath223) [inline README.md:9]: - [ ] You failed to speak about a piece of code you find interesting. Please record an additional loom video talking about this.

Comment (adamteddychang) [reply README.md:9]: Here is the short video for the piece of code I found interesting: 
[Interesting Code Video](https://www.loom.com/share/5b7e7f1321ec444fb6a59a098eedb804)

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @adamteddychang 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**